platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -81,76 +81,77 @@ identity access, infrastructure operations, and app lifecycle.
-include the following:
-
-1. Developers of products and services can automatically provision runtime 
-   capabilities such as compute, storage, databases and service identity and 
-   immediately integrate and use those as part of their products
-1. Developers of products and services can automatically provision supporting 
-   services such as task runners, package registries, deployment orchestrators and 
-   observability systems to build, verify, operate and observe their products
-1. Operators of third-party products and services can automatically provision 
-   spaces and supporting services to deploy and use those third-party products and 
-   services
-
-A more advanced platform also _composes_ these capabilities into experiences and 
-templates fit for key scenarios like web application development or data analysis. 
-For example, an application developer platform can include templates for provisioning 
-complete development environments and could serve the following use cases:
+At its more basic, a platform is an **integrated product** developed by a 
+platform engineering team that is custom built and composed of supporting 
+capabilities and services designed to reduce the cognitive load and burden 
+experienced by **internal** stakeholders of an enterprise organization 
+while still safely shipping software at speed [[4]]. 
+
+Note that while internal stakeholders typically refer to developers, they can 
+refer to DevOps, SREs, FinOps, Customer Success, and anyone who is responsible 
+for software delivery and cloud native infrastructure in an enterprise organization. 
+
+Platforms have become a key focus in cloud-native computing, which typically
+separates supporting capabilities from application-specific logic more than
+previous paradigms. In such environments resources like Kubernetes clusters, 
+databases and object stores, message queues and brokers, observability collectors
+and dashboards,user directories and authentication systems, task runners and reconcilers and
+more are managed independently and integrated into applications running in
+containers and machines. However, this has introduced its own set of challenges as 
+developers for example are having to spend more time learning about infrastructure 
+instead of focusing on software and app delivery.
+
+Therefore, the goal of the platform is to truly drive
+organizational efficiency and productivity by empowering internal stakeholders to 
+focus on their key responsibilities while giving them the tools and self-service 
+necessary to drive their day to day work. For developers for example, this could mean
+having the ability to deploy self-service environments for application development without 
+the overhead of understanding infrastructure components. Or for FinOps, this could
+mean being able to define chargeback rules without understanding getting too in depth of 
+how the metrics are actually collectedfrom cloud-native infrastructure. 
```

Comment:
I found this paragraph a bit confusing and difficult to read in the previous version as well. My concerns are:
1. The number of examples is difficult to parse
2. The use of cloud native here doesn't seem to lead to the challenges proposed.

Here is an attempt to capture what I mean:

```suggestion
It is said that today, every company is a technology company. In addition, the expectations on the availability and scalability of that software are growing. This has led to an increase in pressure on teams to manage more complex infrastructure like queues, high availability data stores, edge caches and more. For additional resiliency and scalability, this infrastructure is mainly built and managed on different physical machines from the software that depends on it. In order to configure and manage these resources developers are having to spend more time learning about infrastructure instead of focusing on software and app delivery.

A platform focuses on centralising the configuration and management of this infrastructure in order to enable organizational efficiency and productivity by empowering internal stakeholders to focus on their key responsibilities while giving application teams the tools and self-service. A team of platform experts not only reduces common work demanded of product teams but also optimizes platform capabilities used in those 
products. A platform team also maintains a set of conventional patterns, knowledge and tools used broadly across the enterprise ; enabling developers to quickly contribute to other teams and products built on the same 
foundations. The shared platform patterns also allow embedding governance and controls in templates, patterns and capabilities. Finally, because platform teams corral providers and provide consistent experiences bespoke to their unique needs over their offerings, they enable efficient use of public clouds and service providers for foundational but undifferentiated capabilities such as databases, identity access, infrastructure operations, and app lifecycle.
```

You will notice I pulled in the last paragraph of the why section as I think that we may want that here instead of in the above section?

Reply:
The implicit question this paragraph is trying to answer is _why_ have "platforms" suddenly become important? The answer posited is that cloud-native computing's distributed systems paradigms lead to a need for platforms of capabilities.

We could make this question and answer more clear, perhaps along the lines of "platforms have become critical for cloud-native computing **because** supporting capabilities are no longer in line with business logic..."

The other items you all mention may fit best in the Attributes section.

Reply:
fixed and included some notes about TVP and how that can evolve into an internal platform